terraform/groups/provisioning-role/module-iam-policies/variables.tf

```diff
@@ -0,0 +1,16 @@
+# AWS
+variable "aws_account_id" {
+  type        = string
+  description = "The AWS account ID."
+}
+
+# IAM
+variable "iam_role_policies" {
+  type        = map
```

Comment:
Could be more specific. E.g. `map(string)`

Reply:
Same in all other places maps are used. I won't comment on all

Reply:
I will check them all. 👍

Reply:
Added variable types to all maps.
Well spotted!